zh/configure-a-tidb-cluster.md

```diff
@@ -534,3 +596,56 @@ affinity:
     ```
 
     其中 `region`、`zone`、`rack`、`kubernetes.io/hostname` 只是举例，要添加的 Label 名字和数量可以任意定义，只要符合规范且和 `pd.config` 里的 `location-labels` 设置的 Labels 保持一致即可。
+
+## 配置平滑升级
```

Comment:
```suggestion
### 配置 TiDB 平滑升级
```

Reply:
Please move this section after the `TiDB service` section。

Reply:
Please also help update the https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-tidb doc.